isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -2501,23 +2500,23 @@ namespace Isis {
                               rotationMatrix[3], rotationMatrix[4], rotationMatrix[5],
                               rotationMatrix[6], rotationMatrix[7], rotationMatrix[8]
                             };
-        m2q_c(CJ, quats[r]);
+        m2q_c(CJ, &quats[r * 4]);
```

Comment:
why multiply the index?

Reply:
In the old code quats was a 2d array, here we changed it to a 1d vector so we have to jump over four elements each time to read the quaternion. I think this largely had to do with 2d vectors not being contiguious